config/module.config.php

```diff
@@ -31,6 +31,12 @@
                 'hydrator_dir'       => 'data/DoctrineMongoODMModule/Hydrator',
                 'hydrator_namespace' => 'DoctrineMongoODMModule\Hydrator',
 
+                'generate_persistent_collections' => Configuration::AUTOGENERATE_ALWAYS,
```

Comment:
I do not believe ZF has any such thing as a "cache-warming process". One would have to generate Doctrine assets manually, which is not a good default for development.